new-frontiers-14/frontier-station-14
# Chemist Wall Lockers. Filled and empty.

Filled and empty chem wall lockers using chem locker inventory. Uses med locker sprite with chem department coloring.




## About the PR



This PR adds an empty and filled chemistry wall locker with updated visual states based on the medical wall locker. The filled inventory is a 1:1 with the standard chemistry locker.

Mentioned by Magnus during a ship walkthrough. So I'm doing my assigned homework.

## Why / Balance


Provides an alternative storage method for departments and expanded options in ship customization.

## How to test


Place lockers. Open/close lockers to test animation states. Check inventory. Check description.

## Media


![ss+(2024-07-05+at+05 31 52)](https://github.com/new-frontiers-14/frontier-station-14/assets/54123313/aaba229f-01e3-4f2f-a647-24ffe774effd)


- [ X ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes


Hopefully none.

**Changelog**


- add: Chemistry wall lockers with corresponding visuals.
